src/components/dagpenger-kalkulator/DagpengerKalkulator.module.scss

```diff
@@ -1,24 +1,78 @@
+@use "styles/common" as common;
+
+$common-border-radius: common.px-to-rem(5px);
+$common-input-max-width: common.px-to-rem(118px);
+
 .container {
   display: flex;
   flex-direction: column;
+  background-color: var(--a-purple-50);
+  border-radius: $common-border-radius;
+  position: relative;
+  padding: common.px-to-rem(80px) 1rem common.px-to-rem(20px);
 
-  .textField {
+  .calculatorFieldset {
+    display: flex;
+    flex-direction: column;
+    border: 0;
+    padding: 0;
+    margin: 0;
+    min-width: 0;
+    width: 100%;
     margin-bottom: 1rem;
-  }
 
-  .radioGroup {
-  }
+    .calculatorTitle {
+      position: absolute;
+      top: 0;
+      left: 0;
+      padding: common.px-to-rem(10px) common.px-to-rem(20px);
+      border-radius: $common-border-radius;
+      background-color: var(--a-purple-200);
+      text-align: center;
+      width: 100%;
+    }
 
-  .select {
-    margin-top: 1rem;
-  }
+    .textField {
+      margin-bottom: 1rem;
+
+      > :global(.navds-text-field__input) {
+        width: auto;
+        max-width: $common-input-max-width;
+      }
+    }
 
-  .button {
-    max-width: fit-content;
-    margin: 1rem auto;
+    .radioGroup {
+      > :global(.navds-radio-buttons) {
+        display: flex;
+        flex-direction: row;
+
+        > :first-child {
+          margin-right: 1rem;
```

Comment:
Har ikke design system egen variabel for den: `a-spacing-x` eller noe sånn?

Reply:
jo, men jeg er litt sånn usikker på om jeg alltid bør bruke dem 🤔 Det er lettere å se at 1rem er 1rem, men det er ikke så lett å se hva f.eks a-spacing-4 er.